Sources/FCM/FCMAndroidConfig/FCMAndroidConfig.swift

```diff
@@ -17,25 +17,25 @@ public struct FCMAndroidConfig: Codable, Equatable {
     /// while 3 seconds and 1 nanosecond should be expressed in JSON format as "3.000000001s".
     /// The ttl will be rounded down to the nearest second.
     /// A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
-    public var ttl: String
+    public var ttl: String?
     
     /// Package name of the application where the registration tokens must match in order to receive the message.
-    public var restricted_package_name: String
+    public var restricted_package_name: String?
     
     /// Arbitrary key/value payload.
     /// If present, it will override FCMMessage.data.
-    public var data: [String: String] = [:]
+    public var data: [String: String]?
     
     /// Notification to send to android devices.
-    public var notification: FCMAndroidNotification
+    public var notification: FCMAndroidNotification?
 
     /// Public Initializer
     public init(collapse_key: String? = nil,
-                priority: FCMAndroidMessagePriority = .normal,
```

Comment:
If we make this `priority` `nil` will it still stay `normal` by default? If yes then ok, if not it may affect current library users who may get unexpected behaviour regarding this `priority` change in their apps in production.

Reply:
If you not sure about that then maybe we could leave this field non-optional and just `normal` by default?

Reply:
I didn't think about the backwards compatibility, but I'm sure the default is normal. You can read about it here: https://firebase.google.com/docs/cloud-messaging/concept-options?hl=nl#ttl

> Normal priority. This is the default priority for data messages.

Here it also indicates that all these fields are optional. Which makes sense, as you can completely leave out the android config and it still sends a notification to Android.

Reply:
Actually it is only the default for data messages, not for other ones, then it is high. I will change this back then.